R/assertions.R

```diff
@@ -270,12 +270,17 @@ assert_character_vector <- function(arg, values = NULL, named = FALSE, optional
 #' Checks if an argument is a logical scalar
 #'
 #' @param arg A function argument to be checked
-#'
-#' @param optional Is the checked argument optional?
-#'
+#' @param optional Is the checked argument optional?\cr
 #' If set to `FALSE` and `arg` is `NULL` then an error is thrown. Otherwise,
 #' `NULL` is considered as valid value.
-#'
+#' @param arg_name (`string`)\cr
```

Comment:
I assume "(`string`)" should state the expected type.

Usually we describe the permitted values after the description of the argument. I am not sure if we should change this.

What is the difference between `string` (here) and `character` (for `message`)?

Reply:
I updated and removed this syntax (that's how NEST projects like it, and i sometimes mix up NEST vs admiral preferences 😆). But for FYI, string is a character vector of length one.

Reply:
> string is a character vector of length one.

Is this a general R rule or a NEST convention?